ebpfcore/ebpf_drv.c

```diff
@@ -406,4 +239,202 @@ _ebpf_driver_query_volume_information(_In_ WDFDEVICE device, _Inout_ IRP* irp)
     irp->IoStatus.Status = status;
     IoCompleteRequest(irp, 0);
     return status;
-}
\ No newline at end of file
+}
+
+static void
+_ebpf_driver_file_close(WDFFILEOBJECT wdf_file_object)
```

Comment:
Line 252 uses `_In_` on a handle and here does not.  If handles are treated as opaque numeric values, don't use `_In_` below, since it's a value type.  If handles are treated as dereferenceable pointers, then use `_In_` since they're reference types.

Reply:
I've fixed one instance in a new function that I'd added (```_ebpf_driver_initialize_device()```).  I've deliberately left other (original) instances (if any) 'as-is' so as to not detract from the primary mods of _this_ PR.

That being said, we can address such issues in the EbpfCore driver codebase in issue #1567.

Reply:
Sounds good